### PR TITLE
Make it compatible for ms1.10.1

### PR DESCRIPTION
**Pull request by SamitHuang**

### CODE REVIEW

File: examples/stable_diffusion_v2/ldm/util.py [hunk 1]
```diff
@@ -55,3 +57,8 @@ def extract_into_tensor(a, t, x_shape):
     b = t.shape[0]
     out = ops.GatherD()(a, -1, t)
     return out.reshape(b, *((1,) * (len(x_shape) - 1)))
+
+def is_old_ms_version(last_old_version="1.10.1"):
```
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/ldm/util.py:61]: mindnlp implements as from _legacy import ops/nn, then nn.Dropout, maybe we follow in later version

File: examples/stable_diffusion_v2/ldm/modules/attention.py [hunk 2]
```diff
@@ -114,7 +114,7 @@ def __init__(self, query_dim, context_dim=None, heads=8, dim_head=64, dropout=1.
         self.to_v = nn.Dense(context_dim, inner_dim, has_bias=False).to_float(dtype)
         self.to_out = nn.SequentialCell(
             nn.Dense(inner_dim, query_dim).to_float(dtype),
-            nn.Dropout(dropout) if version.parse(ms.__version__) < version.parse("1.10") else nn.Dropout(p=1-dropout)
+            nn.Dropout(dropout) if is_old_ms_version() else nn.Dropout(p=1-dropout)
```
Comment (DongyaoZhu) [inline examples/stable_diffusion_v2/ldm/modules/attention.py:117]: 1. for consistency?

**Review by DongyaoZhu — APPROVED** [review]
lgtm

File: examples/stable_diffusion_v2/ldm/util.py (hunk 1 above)
Comment (SamitHuang) [reply examples/stable_diffusion_v2/ldm/util.py:61]: that's another option that requires to wrap all uncompatible ops.
